### PR TITLE
docs: reformat 0001-strategy-for-maintaining-OS-versions.rst

### PR DESCRIPTION
**Pull request by OmarIthawi**
Fix the RST preview on GitHub.

Preview link:

 - https://github.com/Zeit-Labs/openedx-app-ios-contrib/blob/reformat-0001/docs/0001-strategy-for-maintaining-OS-versions.rst

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: docs/0001-strategy-for-maintaining-OS-versions.rst [hunk 1]
```diff
@@ -1,62 +1,68 @@
 Title: Strategy for maintaining iOS versions in the OpenEdx Project
```
Comment (brian-smith-tcril) [inline docs/0001-strategy-for-maintaining-OS-versions.rst:1]: Do we need "Title:" here?

File: docs/0001-strategy-for-maintaining-OS-versions.rst [hunk 2]
```diff
@@ -1,62 +1,68 @@
...
-------
-Enhanced application security.
-Improved performance and functionality.
-Reduced development and testing burden.
-Implementation
+
+- Enhanced application security.
+- Improved performance and functionality.
+- Reduced development and testing burden.
+- Implementation
```
Comment (brian-smith-tcril) [inline docs/0001-strategy-for-maintaining-OS-versions.rst:57]: I don't think "Implementation" is supposed to be a bullet point, it seems like a heading for the next section

**Review by brian-smith-tcril — COMMENTED** [review]
Great formatting changes! Couple tiny notes but this is 99% good to go!

File: docs/0001-strategy-for-maintaining-OS-versions.rst (hunk 1 above)
Comment (OmarIthawi) [reply docs/0001-strategy-for-maintaining-OS-versions.rst]: I don't think so, now removed.

File: docs/0001-strategy-for-maintaining-OS-versions.rst (hunk 2 above)
Comment (OmarIthawi) [reply docs/0001-strategy-for-maintaining-OS-versions.rst]: Yeah, makes total sense. Now correct.

File: docs/0001-strategy-for-maintaining-OS-versions.rst [hunk 3]
```diff
@@ -1,62 +1,70 @@
-Title: Strategy for maintaining iOS versions in the OpenEdx Project
-==================================================
+Strategy for maintaining iOS versions in the OpenEdx Project
```
Comment (brian-smith-tcril) [inline docs/0001-strategy-for-maintaining-OS-versions.rst:1]:
```suggestion
Strategy for maintaining iOS versions in the Open edX Project
```

File: docs/0001-strategy-for-maintaining-OS-versions.rst [hunk 4]
```diff
@@ -1,62 +1,70 @@
...
 
 Status
-------
+******
 Accepted
 
 Context
-------
+*******
 In the OpenEdx project, we are developing a mobile application on the SwiftUI platform for iOS users.
```
Comment (brian-smith-tcril) [inline docs/0001-strategy-for-maintaining-OS-versions.rst:12]:
```suggestion
In the Open edX project, we are developing a mobile application on the SwiftUI platform for iOS users. 
```

**Review by brian-smith-tcril — COMMENTED** [review]
Tiny last thing I didn't catch before.

File: docs/0001-strategy-for-maintaining-OS-versions.rst [hunk 5]
```diff
@@ -1,62 +1,70 @@
-Title: Strategy for maintaining iOS versions in the OpenEdx Project
-==================================================
+Strategy for maintaining iOS versions in the Open edX Project
+############################################################
```
Comment (brian-smith-tcril) [inline docs/0001-strategy-for-maintaining-OS-versions.rst:2]:
```suggestion
#############################################################
```

**Review by brian-smith-tcril — APPROVED** [review]

**Review by volodymyr-chekyrta — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-app-ios&entry.867055334=https://github.com/openedx/openedx-app-ios/pull/378&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2024-03-27+12:41&entry.1917517419=2024-03-28+09:25&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**